### ISSUE TITLE
Extradata with full miner?

### ISSUE THREAD

**Issue by ferrelnoon** [scavicchio/Easy-Walton-Miner#8]
Is it possible to set the extradata with the full miner?

**Comment by scavicchio** [comment]
Yes. You can just add it to the .py file and recompile. I've added an easyMiner extradata in the python code so you can just replace the text with whatever you like.

I'll also post a new version soon that will make it easier to change the extradata.

**Comment by GreatWhite222** [comment]
Hello scavicchio-

Have you completed the Extra Data modification to the program?   Thanks for your efforts!
